packages/logs/src/domain/sender.ts

```diff
@@ -0,0 +1,64 @@
+import type { Context } from '@datadog/browser-core'
+import { combine, createContextManager, display, includes } from '@datadog/browser-core'
+import type { LogsMessage } from './logger'
+import { HandlerType, StatusType } from './logger'
+
+export type Sender = ReturnType<typeof buildSender>
+
+export const STATUS_PRIORITIES: { [key in StatusType]: number } = {
+  [StatusType.debug]: 0,
+  [StatusType.info]: 1,
+  [StatusType.warn]: 2,
+  [StatusType.error]: 3,
+}
+
+export function buildSender(
+  sendLog: (message: LogsMessage) => void,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Naming: ‏`sendLog` is also used as a returned property, with a different meaning. Maybe we can find a more explicit name? Like `sendToHttpImpl`?

Reply:
Sure